lib/gds-components/text-area/TextArea.tsx

```diff
@@ -35,9 +35,9 @@ export const TextArea = ({
         value={value}
         onChange={(e) => onChange(e.target.value)}
       />
-      {hint && (
+      {characterLimit && (
         <div id="more-detail-hint" className="govuk-hint">
-          {hint}
+          You have {characterLimit - value.length} characters remaining
```

Comment:
are we going to use [the GDS component](https://design-system.service.gov.uk/components/character-count/)?

Reply:
forget about my previous comment I was looking only at the latest commit